src/components/ProfilePage/index.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from "react";
+import GamesCarousel from "../GamesCarousel";
+import { useTranslation } from "react-i18next";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
```

Comment:
Since you have imported GamesCarousel too, slick carousel and slick theme are imported at the component, so you may not have to import them.

Reply:
PTAL